exporter/carbonexporter/README.md

```diff
@@ -1,11 +1,15 @@
 # Carbon Exporter
 
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
```

Comment:
@pjanotti, as code owner of this component, do you agree with this [stability level](https://github.com/open-telemetry/opentelemetry-collector#stability-levels)?

Reply:
@djaglowski yes, I agree.